### ISSUE TITLE
Request for Original Data and Guidance on Running Commands

### ISSUE THREAD

**Issue by zengkaipeng** [otori-bird/retrosynthesis#17]
I am inquiring whether you still have the original data for Figure C. I am interested in understanding the benefits of test set augmentation, and for that purpose, I would like to access the original data for the bar labeled "with RSMILES. No test aug" in Figure C.

Additionally, if I wish to perform inference using the checkpoint you provided with the specific settings "with RSMILES, No test aug," could you please guide me on how to organize my running command for this setup?

Your assistance with these requests would be greatly appreciated, and I am looking forward to your response.

![1722873224092](https://github.com/user-attachments/assets/3253254c-aeeb-4f05-94dc-2b954c6131f2)

**Comment by learncode996** [comment]
yeah, I am also interested in the performance of 'No test aug' and wanna quote it in my future work.

**Comment by otori-bird** [comment]
Thanks for your attention to our work.
The original data of top1-top10 in figure C are:
51.192
67.786
75.085
79.124
81.104
81.804
82.324
82.663
82.863
82.973

if you'd perform inference with the specific settings "with RSMILES, No test aug," you coud set the 'augmentation' as 1 and set 'test_only' as true for generate_PtoR_data.py
